lib/api/remoteActions/index.js

```diff
@@ -0,0 +1,77 @@
+/*eslint no-console: 0*/
+var 
+  availableRemoteActions = require('../../config/remoteActions'),
```

Comment:
nitpicking:
Would it be possible to avoid having to maintain an action list as a setting?

Something like this?:

```js
(action, params, args) => {
  if (remoteActions[action] === undefined) {
    try {
      remoteActions[action] = require('./' + action);
    }
    catch(e) {
        console.log('The action "' + action + '" doesn\'t exist');
        return process.exit(1);
    }
  }
}
```